rebar.config

```diff
@@ -1,18 +1,79 @@
-{erl_opts,
- [
-  warn_unused_vars,
-  warn_export_all,
-  warn_shadow_vars,
-  warn_unused_import,
-  warn_unused_function,
-  warn_bif_clash,
-  warn_unused_record,
-  warn_deprecated_function,
-  warn_obsolete_guard,
-  strict_validation,
-  warn_export_vars,
-  warn_exported_vars,
-  warn_missing_spec,
-  warn_untyped_record, debug_info
- ]
-}.
+%% -*- mode: erlang;erlang-indent-level: 2;indent-tabs-mode: nil -*-
+%% ex: ts=4 sw=4 ft=erlang et
+
+%% == Erlang Compiler ==
+
+%% Erlang compiler options
+{erl_opts, [ warn_unused_vars
+           , warnings_as_errors
+           , ewarn_export_all
+           , warn_shadow_vars
+           , warn_unused_import
+           , warn_unused_function
+           , warn_bif_clash
+           , warn_unused_record
+           , warn_deprecated_function
+           , warn_obsolete_guard
+           , strict_validation
+           , warn_export_vars
+           , warn_exported_vars
+           , warn_missing_spec
+           , warn_untyped_record
+           , debug_info
+           ]}.
+
+%% == Dependencies ==
+
+{deps, []}.
```

Comment:
Cowboy should be on this list since we're using `cowboy_req` types in `lasse_handler`.